components/dropdown/dropdown-menu.js

```diff
@@ -12,14 +14,57 @@ import { ThemeMixin } from '../../mixins/theme/theme-mixin.js';
  */
 class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
+	static get properties() {
+		return {
+			_closeRadio: {
+				type: Boolean,
+				reflect: true,
+				attribute: '_close-radio'
+			},
+		};
+	}
+
 	static get styles() {
-		return dropdownContentStyles;
+		return [
+			dropdownContentStyles,
+			css`
+				:host {
+					--d2l-dropdown-close-animation-name: d2l-dropdown-close-animation;
+				}
+
+				:host([theme="dark"]) {
+					--d2l-dropdown-close-animation-name: d2l-dropdown-close-animation-dark;
+				}
+
+				:host([_close-radio]) {
+					animation: var(--d2l-dropdown-close-animation-name) ${dropdownDelay}ms ease-out;
+					animation-delay: 50ms;
+					display: inline-block;
```

Comment:
Something to consider is that if you move the `this.close()` to your animationend event handler (for non-reduced motion), then you can get rid of this and not have style duplication (e.g., if for some reason we were to want to change dropdowns to be something other than `inline-block` in the future, we'd have to know to update here as well).